tools/PI/DevHome.PI/Pages/WERPage.xaml.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using DevHome.Common.Extensions;
+using DevHome.PI.Models;
+using DevHome.PI.Telemetry;
+using DevHome.PI.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Navigation;
+
+namespace DevHome.PI.Pages;
+
+public sealed partial class WERPage : Page
+{
+    private WERPageViewModel ViewModel { get; }
+
+    public WERPage()
+    {
+        ViewModel = Application.Current.GetService<WERPageViewModel>();
+        InitializeComponent();
+    }
+
+    protected override void OnNavigatedTo(NavigationEventArgs e)
+    {
+        base.OnNavigatedTo(e);
+        Application.Current.GetService<TelemetryReporter>().SwitchTo(Feature.WERReports);
+    }
+
+    private void SelectorBar_SelectionChanged(SelectorBar sender, SelectorBarSelectionChangedEventArgs args)
+    {
+        UpdateInfoBox();
+    }
+
+    private void WERDataGrid_SelectionChanged(object sender, SelectionChangedEventArgs e)
+    {
+        UpdateInfoBox();
+    }
+
+    private void UpdateInfoBox()
+    {
+        if (WERDataGrid.SelectedItem is null)
+        {
+            WERInfo.Text = string.Empty;
+            return;
+        }
+
+        SelectorBarItem selectedItem = InfoSelector.SelectedItem;
+        int currentSelectedIndex = InfoSelector.Items.IndexOf(selectedItem);
+        WERDisplayInfo info = (WERDisplayInfo)WERDataGrid.SelectedItem;
+
+        switch (currentSelectedIndex)
+        {
+            case 0: // WER info
+                WERInfo.Text = info.Report.Description;
+                break;
+        }
+    }
+
+    private void WERDataGrid_Sorting(object sender, CommunityToolkit.WinUI.UI.Controls.DataGridColumnEventArgs e)
```

Comment:
Perhaps add a using statement, to avoid these FQNs?

Reply:
Updated. Not sure if it was CoPilot or VS's autocomplete that likes to use the fully qualifed name.